sim/pokemon.js

```diff
@@ -172,10 +211,14 @@ class Pokemon {
 		this.baseHpType = this.hpType;
 		this.baseHpPower = this.hpPower;
 
-		this.clearVolatile(true);
+		this.clearVolatile();
```

Comment:
?

Reply:
The argument (<code>init</code>) wasn't used anywhere in the function so I removed it.